recipe/meta.yaml

```diff
@@ -1,14 +1,15 @@
 {% set name = "py4dstem" %}
-{% set version = "0.11.5" %}
+{% set version = "0.12.10" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/py4DSTEM/py4DSTEM-{{ version }}.tar.gz
-  sha256: 10379c70e4b61171529409a9267ffdf174be851be2fa3bc1a3c7be3b549302c8
+  git_url: https://github.com/py4dstem/py4DSTEM.git
+  git_tag: origin/main
```

Comment:
For reproducibility we want to use only tagged releases: 
```
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  # If getting the source from GitHub, remove the line above,
  # uncomment the line below, and modify as needed. Use releases if available:
  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
  # and otherwise fall back to archive:
  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
  sha256: 2b3a0c466fb4a1014ea131c2b8ea7c519f9278eba73d6fcb361b7bdb4fd494e9
  # sha256 is the preferred checksum -- you can get it for a file with:
  #  `openssl sha256 <file name>`.
  # You may need the openssl package, available on conda-forge:
  #  `conda install openssl -c conda-forge``
```
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml